libs/isograph-react/src/core/entrypoint.ts

```diff
@@ -25,7 +25,7 @@ export type NetworkRequestInfo<TNormalizationAst> = {
 export type IsographEntrypoint<
   TReadFromStore extends { parameters: object; data: object },
   TClientFieldValue,
-  TNormalizationAst = NormalizationAst,
+  TNormalizationAst = NormalizationAst | NormalizationAstLoader,
```

Comment:
`TNormalizationAst extends NormalizationAst | NormalizationAstLoader = NormalizationAst | NormalizationAstLoader`

This way you can't instantiate this with anything unrelated